### DISCUSSION TITLE
Plans for remix-auth 3.6.0+ and remix.run 2.0.0+

### DISCUSSION THREAD

**Discussion by caprica** [pbteja1998/remix-auth-email-link#41] [Q&A]
Is there any intention to refresh this project to support remix-auth 3.6.0+ and remix 2.0.0+?

I get a handful of peer dependency problems as things stand.

**Comment by pbteja1998** [comment]
@caprica please feel free to open a PR with the necessary changes. Happy to review and merge it.

**Comment by pbteja1998** [comment]
@caprica Released it in `v2.1.0`. Please install it and let me know if you face any problems.
